bucket/vdrift.json

```
@@ -6,13 +6,19 @@
     "architecture": {
         "64bit": {
             "url": "https://downloads.sourceforge.net/project/vdrift/vdrift/vdrift-2014-10-20/vdrift-win64-2014-10-20.exe#/dl.7z",
-            "hash": "7149ca6cf41388e29a8a681198fad2452fb4506a1d3404e1dbcb08e9a2c1904d"
+            "hash": "sha1:7a9b4012adb4b31cb83aec58c9eca6c4fbc61b8d"
```

Keep SHA-256 checksum for consistency with other packages:

```suggestion
            "hash": "7149ca6cf41388e29a8a681198fad2452fb4506a1d3404e1dbcb08e9a2c1904d"
```